.gitignore

```diff
@@ -1 +1,2 @@
 /vendor
+.idea
```

Comment:
This should be added to your global gitignore, not per repository. See https://augustl.com/blog/2009/global_gitignores/. 

I'll go ahead and rebase this out in order to get this PR merged.

Reply:
Thanks